pkg/common/agent_grpc.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go-grpc. DO NOT EDIT.
 // versions:
 // - protoc-gen-go-grpc v1.5.1
-// - protoc             v4.25.2
+// - protoc             v5.28.1
```

Comment:
Can regenerate with lower if needed, seems my local is higher?

Reply:
Nice this Pr looks good, and ya if you want to just skip checking in the generated files you can also do that and I can regenerate them when we finish all the proto updates tomorrow